src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,31 @@
+.icon {
+  display: block;
+
+  @include sizeOfIcons($sizeSideOfIconsForMobile);
+
+  @include onTablet() {
+    @include sizeOfIcons($sizeSideOfIconsForTabletAndMore);
+  }
+
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+
+  &--menu {
+    color: $colorBlack;
+
+    background-image: url(../images/BOSE_landing/menu.svg);
+
+    &:hover {
+      background-image: url(../images/menu_hover.svg);
+    }
```

Comment:
There is another icon for hovered menu button in your figma design. And it's black on hover according to design.